labels/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "carto-deckgl-example-labels",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "vite",
+    "dev-local": "vite --config ../vite.config.local.mjs",
```

Comment:
Missing link scripts. You can take them from other examples

Reply:
Added